test/components/stage_progression_button_test.js

```diff
@@ -1,9 +1,12 @@
 import React from "react"
-import { ReactWrapper } from "enzyme"
+import Enzyme from "enzyme"
 import sinon from "sinon"
+import Adapter from "enzyme-adapter-react-16"
 
 import StageProgressionButton from "../../web/static/js/components/stage_progression_button"
 
+Enzyme.configure({ adapter: new Adapter() })
```

Comment:
do we need to do this again here? we already handle this in the `test_helper.j`, right?